pyk/src/pyk/proof/reachability.py

```diff
@@ -502,6 +502,13 @@ def summary(self) -> CompositeSummary:
             ]
         )
 
+    @property
+    def one_line_summary(self) -> str:
+        return (
+            super().one_line_summary
+            + f'/{len(self.kcfg.nodes)} nodes/{len(self.pending)} pending/{len(self.failing)} failing/{len(self.kcfg.ndbranches()) + len(self.kcfg.splits())} branches/{len(self.kcfg.vacuous)} vacuous/{len(self.kcfg.stuck)} stuck'
```

Comment:
I think that we could also add the number of successfully completed branches (nodes that have a cover into target).

Reply:
Added a `passed` section that does this.